public_html/i18n/qqq.json

```diff
@@ -0,0 +1,71 @@
+  "review-by": "Label to indicate which user reviewed a record",
+  "history": "Link to the history page for a given page title\n{{Identical|History}}",
+  "talk": "Link to the talk page for a given page title\n{{Identical|Talk}}",
+  "contribs": "Link to the contributions history for a given page title\n{{Identical|Contributions}}",
+  "record-editcount": "Label to indicate number of edits a user has made",
+  "compare": "Button label to open a panel for text comparison between two pages",
+  "record-ithenticate": "Button label for a button to go to the original plagiarism report generated by Turnitin",
+  "record-url-text": "Label text to show amount of plagiarism.\n Parameters: $1 - Percentage of edit that was plagirized.\n $2 - Number of words plagiarized.",
+  "record-noeditor": "Column text shown to user when no editor was found in database for an edit",
+  "record-noeditor-tooltip": "Tooltip for a text label, shown to editor to explain why no editor could be found",
+  "record-ores": "Ores score for a record. \n Parameters: $1 - Decimal number indicating percentage score.",
+  "article": "Label shown for a text box, showing an article text. \n{{Identical|Article}}",
+  "compare-article-loading": "Temporary text shown to user while article text loads",
+  "compare-source-loading": "Temporary text shown to user while copyvio text source loads",
+  "footer-loadmore": "Label for a button to load more records.",
+  "footer-nomore": "Label to indicate no more records in database.",
+  "footer-broughtby": "Footer text to indicate authors. \n Parameters: $1 - String.",
+  "footer-commtech": "Footer text - name of team.",
+  "footer-poweredby": "Footer text to highlight sources used for development. \n Parameters: $1 - String. $2 - String.",
+  "source": "Footer link to source code for application",
+  "phabricator": "Footer link to phabricator workboard for application",
+  "footer-imageattr": "Footer link to open a pop up box to show attributions for images used in app",
+  "lboard-title": "Page title for leaderboard page",
+  "lboard-week": "Column title for number of records closed last week by each person (top 10 only)",
+  "lboard-month": "Column title for number of records closed last month by each person (top 10 only)",
+  "lboard-alltime": "Column title for the total number of records closed by each person (top 10 only)",
+  "lboard-back": "Button text to go back to CopyPatrol from leaderboard page",
+  "close": "Button text to close pop up box. \n{{Identical|Close}}",
+  "imageattr-title": "Title for popup box to show image attributions",
+  "imageattr-logo": "Title of logo, links to file.",
+  "public-domain": "Text as part of the attributions box",
+  "open-font-license": "Text as part of the attributions box",
+  "imageattr-paragraph1": "Text as part of the attributions box. \n Parameters: $1, $2, $3, $4 - Strings.",
+  "imageattr-paragraph2": "Text as part of the attributions box. \n Parameters: $1, $2, $3 - Strings.",
```

Comment:
We might want to make it clear the imageattr messages should be gender-neutral (User:Xander89 hasn't specified their gender, other proper nouns are not human) and that {{GENDER}} should not be used, since the parameters are links. In Intuition a PHP error is thrown if GENDER is used on markup, I think...